src/main/java/build/buildfarm/common/config/BuildfarmConfigs.java

```diff
@@ -46,4 +55,68 @@ public void loadConfigs(Path configLocation) throws IOException {
       buildfarmConfigs = yaml.load(inputStream);
     }
   }
+
+  public static BuildfarmConfigs loadServerConfigs(String[] args) {
```

Comment:
Seems like there are just a lot of code duplication here. I feel we might want to create some helper functions.